src/bp/ui-shared/webpack.config.js

```diff
@@ -66,6 +66,18 @@ if (process.argv.find(x => x.toLowerCase() === '--analyze')) {
 }
 
 const compiler = webpack(config)
+
+compiler.hooks.done.tap('ExitCodePlugin', stats => {
+  const errors = stats.compilation.errors
+  if (errors && errors.length && process.argv.indexOf('--watch') == -1) {
+    for (const e of errors) {
+      console.log(e.message)
+    }
+    console.log('Webpack build failed')
+    process.exit(1)
```

Comment:
console.error otherwise you will not see the message when using gulp to build (unless you use --verbose)